src/base/memory_monitor_sr.cpp

```diff
@@ -76,7 +91,7 @@ namespace cubperf
   {
     public:
       mmon_component (const char *comp_name)
-	: m_comp_name (comp_name) {}
+	: m_comp_name (comp_name), m_stat {0, 0, 0, 0} {}
```

Comment:
```suggestion
	: m_comp_name {comp_name}, m_stat {0, 0, 0, 0} {}
```
Let's be consistent between () or {}. I'd use {}.

Reply:
Done.